Dockerfile

```diff
@@ -16,7 +16,7 @@ WORKDIR /opt/al_service
 COPY . .
 
 # Patch version in manifest
-ARG version=4.0.0.dev1
+ARG version=4.2.0.dev1
```

Comment:
This is an unnecessary change, this argument is passed in from the build pipeline https://github.com/CybercentreCanada/assemblyline-service-emlparser/blob/60021bd1118d85f8344000e548834567963b78b8/pipelines/azure-build.yaml#L28

Reply:
Unless you build locally and deploy to a fresh microk8s cluster, this value will be read and it'll cause a problem.

Reply:
There is a benefit from the development standpoint otherwise you'd have to build and pass in the arguments as mentioned in the build pipelines.

Reply:
Well it's either we change this value in every single PR we make (assuming 1 PR == 1 dev build), or we don't change this value and it is up to those who are building locally to update it.

Reply:
No, we don't need to match the dev build precisely. We just need to update the minor version (the '2' in 4.2.0.dev1). So we should be good with this version until we get to AL 4.3. I'm not sure if that's what you meant by changing this value in every PR, but it wouldn't be needed.

Reply:
In production, this value makes little difference as it wouldn't break anything and if anything, the system will reject the service update because of a mismatch between the tag and the stated version in the manifest.

The only benefit really comes from creating local builds of the image and pushing to a local registry in a development environment, it's just fewer arguments to pass when you do a `docker build . -t <regisrtry>/<service_name>`.

Reply:
Realistically whenever we encounter a major versioning change, (4.X), we should update the manifests for those services too. But let's face it, it's lower on our todo list than fixing bugs 😉